polaris-migrator/src/migrations/v11-styles-replace-custom-property-depth/v11-styles-replace-custom-property-depth.ts

```diff
@@ -4,7 +4,21 @@ import stylesReplaceCustomProperty from '../styles-replace-custom-property/style
 
 const replacementMaps = {
   '/.+/': {
-    '--p-shadow-transparent': '0 0 0 0 transparent',
+    '--p-shadow-transparent': '--p-shadow-none',
+    '--p-shadow-faint': '--p-shadow-none',
+    '--p-shadow-base': '--p-shadow-xs',
+    '--p-shadow-deep': '--p-shadow-none',
+    '--p-shadow-button': '--p-shadow-sm',
+    '--p-shadow-top-bar': '--p-shadow-sm',
+    '--p-shadow-card': '--p-shadow-md',
+    '--p-shadow-popover': '--p-shadow-xl',
+    '--p-shadow-layer': '--p-shadow-none',
+    '--p-shadow-modal': '--p-shadow-2xl',
+    '--p-shadows-inset-button': '--p-shadow-none',
+    '--p-shadows-inset-button-pressed': '--p-shadow-inset-md',
+    '--p-shadow-color-picker': '--p-shadow-inset-sm',
+    '--p-shadow-color-picker-dragger': '--p-shadow-sm',
+    '--p-hint-from-direct-light': 'rgba(0, 0, 0, 0.15)',
```

Comment:
@heyjoethomas It looks like the last three tokens `--p-shadow-color-picker`, `--p-shadow-color-picker-dragger`, and `--p-hint-from-direct-light` are all being handled in the color migration. However they are being replaced with hard coded values (I'm guessing since the new tokens didn't exist):

| Deprecated Token          | Replacement Value        |
| ------------------------- | ------------------------ |
| `--p-shadow-color-picker` | `rgba(0, 0, 0, 0.5)` |
| `--p-shadow-color-picker-dragger` | `rgba(33, 43, 54, 0.32)` |
| `--p-hint-from-direct-light` | `rgba(0, 0, 0, 0.15)` |